src/Helpers.py

```diff
@@ -155,3 +155,8 @@ def get_items_with_value(world: World, multiworld: MultiWorld, value: str, playe
                             and i.name in world.item_name_groups.get(f'has_{value}_value', [])}
         world.item_values[player][value] = item_with_values
     return world.item_values[player].get(value)
+
+def convertToLongString(input: str | list) -> str: #Todo maybe find a better name for this

```

Comment:
Another minor thing, but function is camel case, but other functions in helpers are lower w/ underscores. So this should also be lower w/ underscores for consistency, since people might also be using the helpers in their hooks.